.idea/clojure-deps.xml

```diff
@@ -3,6 +3,8 @@
   <component name="DepsProjectsManager">
     <option name="projectFiles">
       <list>
+        <option value="file://$PROJECT_DIR$/bridge/deps.edn" />
```

Comment:
I think so. `$PROJECT_DIR$` is not specific to any machine, and this would ensure that `bridge/deps.edn` is loaded as a deps project by any cursive user.